trip.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class trip {
+    private String name;
+    private List<passenger> passengers;
+    private List<place> destinations;
+    private int capacity;
+
+    public trip(String name, int capacity, List<place> destinations){
+        this.name = name;
+        this.passengers = new ArrayList<passenger>();
+        this.destinations = destinations;
+        this.capacity = capacity;
+    }
+
+    public boolean addPassengger(passenger p){
+        if(capacity == passengers.size()) return false;
+        passengers.add(p);
+        return true;
+    }
+
+    public void passengerInformation(){
+        System.out.println("Trip Package name: " + name);
+        System.out.println("Trip Package capacity: " + capacity);
+        System.out.println("Destinations list: ");
+        for(int i=0;i<destinations.size();i++){
+            System.out.println((i+1) + ". " + destinations.get(i));
+        }
+        System.out.println("Passengers Information: ");
+        for(int i=0;i<passengers.size();i++){
+            System.out.print((i+1) + ". ");
+            passengers.get(i).getInformation();
+        }
+    }
+
+    public void packageDetails(){
+        System.out.println("Trip Package name: " + name);
+        System.out.println("Trip Package capacity: " + capacity);
+        System.out.println("Destinations: ");
+        for(int i=0;i<destinations.size();i++){
+            System.out.println((i+1) + ". " + destinations.get(i));
```

Comment:
Does this align with requirements?

Print itinerary of the travel package including:
travel package name,
destinations and details of the activities available at each destination, like name, cost, capacity and description.